src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -71,10 +79,24 @@ public void spinForLow() {
 		m_outtakeSpeedController.set(1.0);
 	}
 
-	public void feed(){
+	public void feedManual(){
 		m_interfaceMotor.set(m_interfaceSpeed);
 	}
 
+	public void feedTime() {
+		if (feedWaitTime != 0.0 && m_clock.getTime() > feedWaitTime) {
+			m_interfaceMotor.set(m_interfaceSpeed);
+			feedWaitTime = 0.0;
+		}
+		else if (feedStartTime != 0.0) {
```

Comment:
is this supposed to be feedStartTime == 0.0 ?

Running through this logic, I don't see a way one of these if statements executes without this change.